packages/build-info/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@netlify/build-info",
+  "version": "4.0.10",
+  "description": "Build info utility",
+  "type": "module",
+  "exports": "./lib/main.js",
+  "main": "./lib/main.js",
+  "types": "./lib/main.d.ts",
+  "bin": {
+    "build-info": "./bin.js"
+  },
+  "files": [
+    "bin.js",
+    "lib/**/*"
+  ],
+  "scripts": {
+    "prebuild": "rm -rf lib",
+    "build": "tsc",
+    "test": "ava",
+    "test:ci": "c8 -r lcovonly -r text -r json ava"
+  },
+  "ava": {
```

Comment:
Extend Ava's base config.

Reply:
In this case we don't want to extend ava's base config as all the properties are different and it's a different project basically. So testing was set up differently. (No splitting up needed here etc.)

Reply:
Ah I see. We should at least use **ava.config.js** instead of having the configuration in package.json that way this package will be structurally consistent with the other packages.